proposals/legacy-and-deprecated-data-elements.md

```diff
@@ -0,0 +1,259 @@
+
+<br /><br />
+
+# RESO End User License Agreement (EULA)
+
+[**This End User License Agreement (the "EULA")**](https://www.reso.org/eula/) is entered into by and between the Real Estate Standards Organization ("RESO") and the person or entity ("End User") that is downloading or otherwise obtaining the product associated with this EULA ("RESO Product"). This EULA governs End Users use of the RESO Product and End User agrees to the terms of this EULA by downloading or otherwise obtaining or using the RESO Product.
+
+The keywords "MUST", "MUST NOT", "REQUIRED", "SHALL", "SHALL NOT", "SHOULD", "SHOULD NOT", "RECOMMENDED", "MAY", and "OPTIONAL" in this document are to be interpreted as described in [RFC 2119](https://www.ietf.org/rfc/rfc2119.txt).
+
+<br />
+
+# Table of Contents
+- [Introduction](#introduction)
+- [Section 1: Purpose](#section-1-purpose)
+- [Section 2: Specification](#section-2-specification)
+- [Section 3: Certification](#section-3-certification)
+- [Section 4: Contributors](#section-4-contributors)
+- [Section 5: References](#section-5-references)
+- [Section 6: Appendices](#section-6-appendices)
+- [Section 7: License](#section-7-license)
+
+# Introduction
+
+This specification extends the Data Dictionary Field and Lookup resource to convey information about new and deprecated models, fields, and lookups.
+
+# Section 1: Purpose
+
+Over time the fields and related lookup values in use by an MLS organization evolve and change. In the 2000's, the field **BusinessType** having a lookup value of **Video Store** was applicable. In the 2020's, video stores have become an echo of the past. Our data in the MLS world though still contains historical listing data where that lookup choice is correct. In our systems today, we may not allow a new listing to use that lookup choice, but we want to inform the data consumers that it was a valid choice at one time and may appear in older data.
+
+Fields as well evolve over time. Whether it is a field that has been retired from use or a field that has been renamed to a Data Dictionary compliant name. It is useful for an MLS organization to be able to convey to data consumers when these changes happen. 
+
+This specification introduces additions to the [Field](https://ddwiki.reso.org/display/DDW17/Field+Resource), [Lookup](https://ddwiki.reso.org/display/DDW17/Lookup+Resource), and upcoming Model resources to allow this information to be conveyed.
+
+# Section 2: Specification
+
+## Field Resource
+
+The Data Dictionary 1.7 specification defines the **Field** Resource with the following fields:
+
+- **DisplayName**
+- **FieldKey**
+- **FieldName**
+- **ModificationTimestamp**
+- **ResourceName**
+
+_Note: ResourceName will be deprecated in favor of ModelKey starting in Data Dictionary 2.1._
+
+The following fields are proposed as additions:
+
+- **FieldStatus**: closed enumeration with allowed values: **Obsolete**, **Deprecated**, **Active**, **Transition**
```

Comment:
Note: there is a pending question as to whether we use something more general here, like ElementStatus, rather than Model/Field/LookupStatus.